applications/openshift/worker/file_owner_worker_ca/rule.yml

```diff
@@ -18,6 +18,8 @@ identifiers:
     cce@ocp4: CCE-83495-2
 
 references:
+    bsi: APP.4.4.A17
+    bsi: APP.4.4.A17
```

Comment:
the reference is doubled

Reply:
Thanks, I will resolve it in the next commit